packages/core/src/messages/responses/enums/index.ts

```diff
@@ -0,0 +1,28 @@
+export enum MomentoResponse {
```

Comment:
we should name this something else that is not `Response` since node has a default response object, and it confuses the "compiler"